octopoes/octopoes/xtdb/client.py

```diff
@@ -59,7 +59,7 @@ def _verify_response(response: Response) -> None:
         try:
             response.raise_for_status()
         except HTTPStatusError as e:
-            logger.error(e.response.status_code)
+            logger.debug(e)
             if e.response.status_code != codes.NOT_FOUND:
                 logger.error(response.request.url)
```

Comment:
I think we still want to log with level error when the status code isn't 404? Else we just log the request url en content if debug isn't enabled which isn't very useful.

Reply:
This was the previous behavior https://github.com/minvws/nl-kat-coordination/pull/2795/files#diff-1f4c0c32e0407dc2b0e94a92c5bbd9979d8a810c4d980a10769be9f2ad49565b. The logging was actually useful for creating that patch -- to understand all the various ways that errors can occur.

Either is fine with me.

Reply:
Agree with both; logging the entire response context isn't appropriate in this case, and logging the status error (code or code/ reason combination) is more meaningful (and preferably with some text such that we can differentiate between request/ response logs)

Reply:
Another try 47194be7a5ad76861245623ebb914c72a43f0a7c

Reply:
Another try in a22ef2a7ad9770c44b3dfa757fc768effe88ac31

Reply:
Resolved in 8186b8ceb596dd36b96954424dbc76a03a5f9e67